examples/granite_example/readme.md

```diff
@@ -0,0 +1,24 @@
+# InternVL Inference
+This directory contains an example script of how to run inference on Granite-vision-3.2-2b via QEFFAutoModelForCausalLM class.
+
+Currently for this model we will support dual pcs. No CB support is there for this model.
+
+The model expects the following inputs to be fixed. Please reshape any given image to (w x h) (1610 x 1109) and than pass it to the processor.It accepts a path or a url.
```

Comment:
nit: did we add small script or change in example to do this reshape? how ill customer know this?